src/app/store/reducers/ReferenceData.reducers.ts

```diff
@@ -0,0 +1,29 @@
+import { initialReferenceDataState } from '@app/store/state/ReferenceDataState.state';
+import {
+  EReferenceDataActions,
+  ReferenceDataActions
+} from '@app/store/actions/ReferenceData.actions';
+
+export function ReferenceDataReducers(
+  state = initialReferenceDataState,
+  action: ReferenceDataActions
+) {
+  switch (action.type) {
```

Comment:
up to you again but object destructuring will make things clearer I think.
```
const {type, preparers, testStations} = action

// then in the reducers we would even simply things like
case EReferenceDataActions.LoadPreparersSuccess: {
  ({ // <-- this is the same as returning an object with curly braces, not sure about the linter rules though
     ...state,
     preparers
   })
}
...

```

Reply:
As agreed as part of this PR, we will add a technical task to refactor the state variables which was done prior to ADR:

Reference ticket: https://jira.dvsacloud.uk/browse/CVSB-15098